src/routes/calculator/_fateCount.svelte

```diff
@@ -0,0 +1,155 @@
+          totalPrimogem += total
+          result.push({name: value.name, image: value.image, amount: value.amount, total: total })
+        }
+      }
+    }
+    if (parameters[0].amount>0) {
+      let total = parameters[0].amount*60;
+      result.push({name: $t('calculator.fateCount.dailyCommission'), image: "/images/commission.png", amount: parameters[0].amount, total: total})
+    }
+  }
+
+  function onChange() {
+    result = null;
+    totalPrimogem = 0;
+  }
+</script>
+
+<div class="bg-item rounded-xl p-4">
+  <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-2 xl:grid-cols-3 gap-4">
+    <div>
+      <div>
+        {#each fateValues as value}
+          <div class="mb-1 rounded-xl border-background border p-2">
+            <div class="flex flex-row items-center mb-1">
+              <img src={value.image} alt={value.name} class="w-6 mr-2" />
+              <p class="text-white">{value.name}</p>
+            </div>
+            <div class="flex flex-row items-center">
+              <div class="w-full">
+                <Input className="text-center" bind:value={value.amount} on:change={onChange} type="number" min="0" step="1" pattern="\d*" />
+              </div>
+            </div>
+          </div>
+        {/each}
+      </div>
+    </div>
+    <div>
+      {#each parameters as value}
+        <div class="mb-1 rounded-xl border-background border p-2">
+          <div class="flex flex-row items-center mb-1">
+            <p class="text-white">{value.name}</p>
+          </div>
+          <div class="flex flex-row items-center">
+            <div class="w-full">
+              <Input className="text-center" bind:value={value.amount} on:change={onChange} type="number" min="0" step="1" pattern="\d*" />
+            </div>
+          </div>
+        </div>
+      {/each}
+    </div>
+    <div class="md:col-span-2 xl:col-span-1">
+      <Button className="block w-full md:w-auto" on:click={calculate}
+        >{$t('calculator.fateCount.calculate')}</Button
+      >
+      {#if result !== null}
+        <script>console.log(result);</script>
+        <div>
+          <div
+          transition:fade={{ duration: 100 }}
+          class="rounded-xl bg-background p-4 block md:inline-block"
+          style="height: fit-content; width: fit-content;"
+          >
+            <table>
+              <tr>
+                <th class="text-white pr-2 text-center">{$t('calculator.fateCount.type')}</th>
+                <th class="text-white pr-2 text-center">{$t('calculator.fateCount.amount')}</th>
+                <th class="text-white pr-2 text-right">{$t('calculator.fateCount.totalPrimogem')}</th>
+              </tr>
+              {#each result as res}
+                <tr>
+                  <td class="border-t border-gray-700 text-white pr-2">
+                    <span class="inline-flex align-middle">
+                      {#if res.image !== null}<img src={res.image} alt={res.name} class="w-6 mr-2" />{/if}
```

Comment:
![image](https://user-images.githubusercontent.com/2796366/149627438-48c31d9a-f0c9-428b-8d99-475f94673ba8.png)
limit the icon height

Reply:
done